tests/structured_conf/test_structured_config.py

```diff
@@ -1144,3 +1144,191 @@ def test_merge_missing_list_promotes_target_type(self, class_type: str) -> None:
         c3 = OmegaConf.merge(c1, c2)
         with pytest.raises(ValidationError):
             c3.missing.append("xx")
+
+    @pytest.mark.parametrize(
```

Comment:
This is a ton of new tests, good job.
Move them to a dedicated file though.

`test_structured_config_container_element.py`?